js/binance.js

```diff
@@ -1412,14 +1429,24 @@ module.exports = class binance extends Exchange {
                 if (isDefault || (fee === undefined)) {
                     fee = withdrawFee;
                 }
+                const precisionTick = this.safeString (networkItem, 'withdrawIntegerMultiple');
+                // avoid zero values, which are mostly from fiat or leveraged tokens : https://github.com/ccxt/ccxt/pull/14902#issuecomment-1271636731
+                // so, when there is zero instead of i.e. 0.001, then we skip those cases, because we don't know the precision - it might be because of network is suspended or other reasons
+                if (!Precise.stringEq (precisionTick, '0')) {
+                    minPrecision = (minPrecision === undefined) ? precisionTick : Precise.stringMin (minPrecision, precisionTick);
+                }
```

Comment:
I think we could simplify this a bit by replacing ⬆️ with ⬇️ 
```Javascript
                const precisionTick = this.omitZero(this.safeString (networkItem, 'withdrawIntegerMultiple'));
                minPrecision =  Precise.stringMin (minPrecision, precisionTick);
```

Reply:
Precise.X handles `undefined` values well btw

Reply:
Sorry Carlos, I don't think above suggestion will work, because `Precise.stringMin/Max` will fail the logic if argument is `undefined` - https://github.com/ccxt/ccxt/blob/master/js/base/Precise.js#L241
as you see, when one of the argument is `undefined` it returns `undefined`, which will crash the above logic. lmk if I miss smth.

Reply:
@ttodua sorry u're right yeah let's keep it as it is